ext/node/ops/os/mod.rs

```diff
@@ -75,6 +75,25 @@ where
   Ok(euid)
 }
 
+#[op2(fast)]
+pub fn op_getegid<P>(state: &mut OpState) -> Result<u32, AnyError>
+where
+  P: NodePermissions + 'static,
+{
+  {
+    let permissions = state.borrow_mut::<P>();
+    permissions.check_sys("getegid", "node:os.getegid()")?;
```

Comment:
Add `getegid` in https://github.com/denoland/deno/blob/147411e64b22fe74cb258125acab83f9182c9f81/runtime/permissions/lib.rs#L915-L918